source/includes/_async_jobs.md

```diff
@@ -0,0 +1,156 @@
+
+```shell
+# EXAMPLE REQUEST
+$ curl "https://app.conversio.com/api/v1/async-jobs" \
+  -H "X-ApiKey: YOUR_API_KEY" \
+  -H "Accept: application/json" \
+  -X GET
+```
+
+> EXAMPLE RESPONSE
+
+```json
+{
+  "data": [
+    {
+      "id": "57b5aa3b046abfb053d80b52",
+      "kind": "newsletter-recipients",
+      "status": "done",
+      "startedAt": "2017-01-03T16:32:27.741Z",
+      "completedAt": "2017-01-03T16:37:20.417Z",
+      "result": "https://app.conversio.com/async-jobs/57b5aa3b046abfb053d80b52.csv"
+    },
+    {
+      "id": "57b5aa3b046abfb053d80b54",
+      "kind": "newsletter-recipients",
+      "status": "pending"
+    },
+    {
+      "id": "57b5aa3b046abfb053d80b57",
+      "kind": "newsletter-recipients",
+      "status": "failed",
+      "startedAt": "2017-01-02T05:33:22.123Z",
+      "completedAt": "2017-01-02T05:33:45.412Z",
+      "error": "Something terrible as happened. Get in touch with support tout de suite!"
+    }
+  ]
+}
+```
+
+### List all Jobs [GET]
+
+`https://app.conversio.com/api/v1/async-jobs`
+
+_OAuth Scopes_: read_async_jobs
+
+<aside class="success">
+  Response 200 (application/json)
+</aside>
+
+### Response Body
+
+Then endpoint returns an object with a `data` key that is an Array with all the async jobs in the shop. It will limit output to the currently authenticated Partner App webhooks if authorized through OAuth.
+
+Each Async Job includes the following info:
+
+|Key               |Details    |
+|-----------------:|-----------|
+|**id:**           |**string**|
+|                  |The Async Job ID. Use it when calling single-job endpoints.|
+|**kind:**         |**string**|
+|                  |What kind of job this is. Currently only supports "newsletters-recipients".|
+|**status:**       |**string**|
+|                  |The job's status. One of "pending", "done" or "failed".|
+|**startedAt:**    |**string, optional**|
+|                  |When this job started processing. Is an **ISO 8601** encoded date. Can be null if it hasn't started yet.|
+|**completedAt:**  |**string, optional**|
+|                  |When this job completed. There's a `result` or `error` if this is present. Is an **ISO 8601** encoded date. Can be null if it hasn't finished yet.|
+|**error:**        |**string, optional**|
+|                  |An error message that indicates a problem when processing the job. There is no `result` if this is present.|
+|**result:**       |**string, optional**|
+|                  |The final result from processing this job. What it is depends on job `kind`.|
+|**partnerApp:**   |**string, optional**|
+|                  |The ID of the Partner App that created this Async Job. Included only when authorization is made with the API key.|
+
+## Get Async Job
+
+Returns the requested async job. When using OAuth, only jobs started by the authenticated app can be accessed.
```

Comment:
Should 'async job' be 'Async Job' just to keep it consistent with the rest of the references to this?